doc/contribute/eclipse_workflow.rst

```diff
@@ -0,0 +1,346 @@
+Developing NEST with Eclipse
```

Comment:
I nominate @heplesser to review the Eclipse Development documentation, as he is actually using it.